main/src/ca/uwaterloo/flix/language/phase/Parser2.scala

```diff
@@ -1651,6 +1659,7 @@ object Parser2 {
         case TokenKind.KeywordUnsafe => unsafeExpr()
         case TokenKind.KeywordUnsafely => unsafelyRunExpr()
         case TokenKind.KeywordRun => runExpr()
+        case TokenKind.KeywordHandler => handlerExpr()
```

Comment:
This list should be alphabetical

Reply:
It is very much not alphabetical - i don't know if the ordering is intentional?

Reply:
I see. What a mess. We can leave it as is -- in this PR.